src/System.CommandLine/Parsing/Token.cs

```diff
@@ -59,21 +59,5 @@ internal Token(string? value, TokenType type, Symbol? symbol, int position)
 
         /// <inheritdoc />
         public override string ToString() => Value;
-
```

Comment:
Standard guidance (https://learn.microsoft.com/en-us/dotnet/api/system.iequatable-1?view=netstandard-2.0#notes-to-implementers) does suggest we should keep the equality operators for consistency, even if we aren't using them at the moment.

From a personal perspective, I like having the equality operators available since it makes the code simpler when dealing with nullable types, and using the equality operators helps avoid (via compilation errors) accidentally making typos that result in calling `type1.Equals(type2)` (calling into `Type1.Equals(object)`, always returning false).